src/main/scala/com/johnsnowlabs/ml/ai/Bert.scala

```diff
@@ -57,6 +62,10 @@ private[johnsnowlabs] class Bert(
     extends Serializable {
 
   val _tfBertSignatures: Map[String, String] = signatures.getOrElse(ModelSignatureManager.apply())
+  val detectedEngine: String =
+    if (tensorflowWrapper.isDefined) ModelEngine.tensorflow
+    else if (onnxWrapper.isDefined) ModelEngine.onnx
+    else ModelEngine.tensorflow
```

Comment:
By default, it's going to be TensorFlow. Why is ONNX, not the default?

Reply:
If there is model that doesn't have model's engine is definitely a `tensorflow` since those are existing models before we added this feature. (maybe around 17k models already that don't have this param)